playbooks/roles/document_hosting_k8s/defaults/main.yaml

```diff
@@ -1,6 +1,7 @@
 document_hosting:
   swift_server: swift.example.com:443
   server_name: docs.example.com
+  beta_server_name: docs-beta.example.com
```

Comment:
I guess we should better have something like "alt" - a general way for alternate or synonyms. Eventually we can convert server_name into domains (an array)

Reply:
the array is less transparent than 2 specific servernames

Reply:
issue is that from the webserver pov you can have as many hostnames as you wish